autopilot/logging.go

```diff
@@ -0,0 +1,46 @@
+package autopilot
+
+import (
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func newLogger(path string) (*zap.Logger, error) {
+	file, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
Who is responsible for closing the file? We need to make sure it's closed when we shut things down. Otherwise Windows might end up being unhappy again.

Reply:
Turns out it was unhappy. I added `onStopFns` to the autopilot in which I sync the logger and close the file.